docs/configuration.md

```diff
@@ -65,10 +79,9 @@ export default defineConfig({
 });
 ```
 
-Viewport sizes can range from `300x300` to `2000x2000` for Chrome and Firefox.
-Edge and Safari must be between `400x400` and `1200x1200`. The `ios-safari`
-target runs on an iPhone with a fixed viewport of `375x667`. The `ipad-safari`
-target is always `1080x810`.
+Viewport sizes can range from `300x300` to `2000x2000`. The `ios-safari` target
+runs on an iPhone with a fixed viewport of `375x667`. The `ipad-safari` target
+is always `1080x810`.
```

Comment:
Here we should probably say `browserType` instead of `target`

Reply:
Or now that we changed browserType to type, maybe target type is the best phrasing?